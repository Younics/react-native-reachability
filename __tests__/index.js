jest.setMock('react-native', {
  NativeModules: {
    RNReachability: {
      isReachable: jest.fn().mockResolvedValue(1)
    }
  }
})

describe('Network', () => {
  const Network = require('../index')

  it('resolves a promise', () => {
    expect.assertions(1)
    return expect(Network.isReachable()).resolves.toBe(true)
  })
})
