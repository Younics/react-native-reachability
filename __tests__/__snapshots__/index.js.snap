// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Network component renders without throwing 1`] = `null`;
